sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/IVirtualMachineScaleSetsOperations.cs

```diff
@@ -153,7 +157,7 @@ public partial interface IVirtualMachineScaleSetsOperations
         /// <exception cref="Microsoft.Rest.ValidationException">
         /// Thrown when a required parameter is null
         /// </exception>
-        Task<AzureOperationResponse<VirtualMachineScaleSet>> GetWithHttpMessagesAsync(string resourceGroupName, string vmScaleSetName, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken));
+        Task<AzureOperationResponse<VirtualMachineScaleSet>> GetWithHttpMessagesAsync(string resourceGroupName, string vmScaleSetName, string expand = default(string), Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken));
```

Comment:
This change requires a [Customization ](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/compute/Microsoft.Azure.Management.Compute/src/Customizations) file. 

you will need to create a Customization file for this issue.
1 file in the Customization folder for each file in this PR that has a new parameter (not the last parameter) in the method signature.
Each Customization file needs these:

1) 1 method with each parameter up to but not including the new parameter.
2) 1 method for each parameter that is after the new parameter.
This means 1 new method for each parameter up to the parameter 'customHeaders' NOT including the new parameter.
Then 1 new method for each parameter up to 'cancellationToken' NOT including the new parameter.
Do this until you reach the last parameter.

Reply:
thank you for the explanation, but sorry, I'm having trouble understanding these directions.
Is there a wiki or some documentation that also explaints this?

you said: This means 1 new method for each parameter up to...
so if a method has ten parameters, does that means I would need ~ten new methods?

Reply:
can we use a different example:
Let's say we have this method
Method(a, b, c, d, e, f)
now i am adding a new param h between d and e
Method(a, b, c, d, h, e, f)

what would the customization file require?